desktop/src/main/java/bisq/desktop/main/dao/economy/supply/SupplyView.java

```diff
@@ -291,85 +298,75 @@ private void createSupplyLockedInformation() {
-            XYChart.Series<Number, Number> seriesBSQIssuedMonthly,
-            XYChart.Series<Number, Number> seriesBSQBurntMonthly
+        XYChart.Series<Number, Number> seriesBSQIssuedMonthly,
+        XYChart.Series<Number, Number> seriesBSQBurntMonthly
     ) {
-        Supplier<NumberAxis> makeXAxis = () -> {
-            NumberAxis xAxis = new NumberAxis();
-            configureAxis(xAxis);
-            xAxis.setTickLabelFormatter(getTimestampTickLabelFormatter("MMM uu"));
-            return xAxis;
-        };
+        xAxisChart1 = new NumberAxis();
+        configureAxis(xAxisChart1);
+        xAxisChart1.setLabel("Month");
+        xAxisChart1.setTickLabelFormatter(getMonthTickLabelFormatter("MM\nyyyy"));
+        addTickMarkLabelCssClass(xAxisChart1, "axis-tick-mark-text-node");
 
-        Supplier<NumberAxis> makeYAxis = () -> {
-            NumberAxis yAxis = new NumberAxis();
-            configureYAxis(yAxis);
-            yAxis.setTickLabelFormatter(BSQPriceTickLabelFormatter);
-            return yAxis;
-        };
-
-        var chart = new LineChart<>(makeXAxis.get(), makeYAxis.get());
+        NumberAxis yAxis = new NumberAxis();
+        configureYAxis(yAxis);
+        yAxis.setLabel("BSQ");
+        yAxis.setTickLabelFormatter(BSQPriceTickLabelFormatter);
 
+        var chart = new LineChart<>(xAxisChart1, yAxis);
         configureChart(chart);
-        chart.setCreateSymbols(false);
-
-        chart.getData().addAll(List.of(seriesBSQIssuedMonthly, seriesBSQBurntMonthly));
-
         chart.setLegendVisible(true);
+        chart.setCreateSymbols(true);
+
+        chart.getData().addAll(seriesBSQIssuedMonthly, seriesBSQBurntMonthly);
 
         return chart;
     }
 
-    private Node createBSQIssuedChart(XYChart.Series<Number, Number> series) {
-        NumberAxis xAxis = new NumberAxis();
-        configureAxis(xAxis);
-        xAxis.setTickLabelFormatter(getTimestampTickLabelFormatter("MMM uu"));
+    // chart #2 (middle) - removed
+    // private Node createBSQIssuedChart(XYChart.Series<Number, Number> series) {
+    //     NumberAxis xAxis = new NumberAxis();
+    //     configureAxis(xAxis);
+    //     xAxis.setTickLabelFormatter(getTimestampTickLabelFormatter("MMM uu"));
 
-        NumberAxis yAxis = new NumberAxis();
-        configureYAxis(yAxis);
-        yAxis.setTickLabelFormatter(BSQPriceTickLabelFormatter);
+    //     NumberAxis yAxis = new NumberAxis();
+    //     configureYAxis(yAxis);
+    //     yAxis.setTickLabelFormatter(BSQPriceTickLabelFormatter);
 
-        AreaChart<Number, Number> chart = new AreaChart<>(xAxis, yAxis);
-
-        configureChart(chart);
-        chart.setCreateSymbols(false);
+    //     AreaChart<Number, Number> chart = new AreaChart<>(xAxis, yAxis);
+    //     configureChart(chart);
+    //     chart.setCreateSymbols(true);
 
-        chart.getData().add(series);
+    //     chart.getData().add(series);
 
-        return chart;
-    }
+    //     return chart;
+    // }
```

Comment:
If you want to remove it just get rid of it completely 😄

Reply:
OK. Done.
Thanks for the input. Hesitated to do by myself.